### ISSUE TITLE
Step 1.2: GSM and GLEAM

### ISSUE THREAD

**Issue by steven-murray** [HERA-Team/hera-validation#40]
This test carries on the idea of 1.0 and 1.1 to add foregrounds. In this case, it is the simple extension to have *both* point-source and diffuse foregrounds along with the EoR. It is reasonably to use a flat-spectrum EoR in this case, as it is not really the shape-recovery of P(k) under question.

* Simulation Component: GSM, GLEAM, EoR (flat)
* Simulators: `RIMEz`
* Pipeline Components: `pspec`
* Depends on: 1.0 and 1.1

## Why this test is required

It is unlikely, but there may be increased complexity in the P(k) recover from having both point source and diffuse foregrounds. Furthermore, having both of these components is important for tests down the line.

## Summary
A brief step-by-step description of the proposed test follows:

* Obtain simulated visibilities of GSM and GLEAM from 1.0 and 1.1 respectively.
* Use `pspec` to obtain P(k)

## Simulation Details

* Freq. range:
* Channel width:
* Baseline/antenna configuration:
* Total integration time:
* Number of realisations:

## Criteria for Success

* EoR P(k) outside wedge is recovered to within 1%

**Comment by r-pascua** [comment]
Is there a reason why we would *prefer* to use a flat P(k) rather than a power-law P(k)? Mainly asking because, as far as I know, the flat P(k) simulation available on `lustre` is over a smaller bandwidth and with a much smaller set of baselines (only three) than the power-law P(k) simulation. If we're okay using a power-law EoR P(k) rather than a flat-spectrum EoR signal, then this test will be much easier to run, as all of the processed data is actually available already (it was produced during the analysis for #32).

**Comment by steven-murray** [comment]
No, I don't think there's a good reason -- it was chosen a long time ago. I'll change it!

**Comment by steven-murray** [comment]
So this can be done almost straight away then, right?

**Comment by r-pascua** [comment]
In #32, there is wiggling/jagging in the unperturbed power spectrum estimate (i.e. what will be investigated in this test). This test should experiment with `pspec` settings to determine the source of this behavior.
